src/lib.rs

```diff
@@ -169,6 +171,18 @@ impl Blockchain {
         let tx = psbt.internal.lock().unwrap().clone().extract_tx();
         self.get_blockchain().broadcast(&tx)
     }
+
+    fn get_height(&self) -> Result<u32, Error> {
+        return self.get_blockchain().get_height();
+    }
+
+    fn get_block_hash(&self, height: u32) -> Result<String, Error> {
+        let block_hash = self
```

Comment:
This is a good place to use `.map()` and also you won't need the `.unwrap()`. It should look something like this:
```suggestion
             self.get_blockchain()
            .get_block_hash(u64::from(height))
            .map(|hash| hash.to_string())
```

Reply:
Yep sounds good!